### DISCUSSION TITLE
User Registration

### DISCUSSION THREAD

**Discussion by Mzeey** [mzeey-tutorials/NepaApp#86] [Process Flows]
# Process Flow: User Registration

## Description
This process flow illustrates the steps involved in the user registration process.

## Process Overview
The user registration process is designed to allow new users to create accounts and gain access to the system.

## Diagram
Link to Miro Board: [User Registration Process](https://miro.com/app/board/uXjVNOFF4QE=/?moveToWidget=3458764573281081802&cot=14)

![Screenshot 2023-12-14 231633](https://github.com/mzeey-tutorials/NepaApp/assets/92148086/66e215fc-39a6-4c8f-bd9f-f0df709812e6)

## Notes
- This process heavily relies on email verification to ensure account security.
- Users who have not verified their email can take additional actions, such as requesting a new verification link or seeking support.

**Comment by devshalem** [comment]
Greetings bro.. are we moving to the next iteration?

**Reply by Mzeey** [reply]
My apologies for the late reply, the iteration move automatically, after it's assigned date has expired
